app/code/core/Mage/Rule/Model/Condition/Combine.php

```diff
@@ -255,8 +255,8 @@ public function asXml($containerKey = 'conditions', $itemKey = 'condition')
      */
     public function loadArray($arr, $key = 'conditions')
     {
-        $this->setAggregator($arr['aggregator'] ?? ($arr['attribute'] ?? null))
-            ->setValue($arr['value'] ?? ($arr['operator'] ?? null));
+        $this->setAggregator($arr['aggregator'] ?? $arr['attribute'] ?? null)
+            ->setValue($arr['value'] ?? $arr['operator'] ?? null);
```

Comment:
IMHO this is hard to understard, extremely short code is already less easy on the brain, but this (to me) it's a no. with the parenthesis was a bit better

Reply:
imho ... for me its easier to read w/o parenthesis.  I can revert it later if you think it was better  before.

Reply:
Ref https://www.php.net/manual/en/migration70.new-features.php, the operator can be chained. With parenthesis, it defeats the feature.

Reply:
Chaining seems to me a good argument against the additional parentheses.